client/service-worker.js

```diff
@@ -0,0 +1,41 @@
+// The Lounge - https://github.com/thelounge/lounge
+/* global clients */
+"use strict";
+
+self.addEventListener("push", function(event) {
```

Comment:
Well, for 1 you just said earlier that you don't know if all browsers support arrow functions.

The thing is, that ES6 isn't a binary thing. Plenty of browsers support some features and not others. And transpiling is for future things as well. 

I don't really care enough, I was just giving you a solution for the problem you were discussing.